learning_wavelets/models/exact_recon_unet.py

```diff
@@ -139,7 +153,10 @@ def call(self, inputs):
         outputs = self.final_conv(outputs)
         noise_std = tf.reshape(noise_std, shape=[tf.shape(noise_std)[0], 1, 1, 1])
         outputs = tf.image.resize_with_crop_or_pad(outputs, h, w)
-        outputs = noisy_image - noise_std * outputs
+        if self.exact_recon:
+            outputs = noisy_image - noise_std * outputs
+        else:
+        	outputs = noisy_image - outputs
```

Comment:
watch out here: let's make sure we only use spaces and not tabulations for the whitespaces